src/main/webapp/user-page.html

```diff
@@ -21,7 +21,7 @@
     <meta charset="UTF-8">
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
-    <script src="/js/user-page-loader.js"></script>
+    git c
```

Comment:
Do we need this?

Reply:
I also don't know if we want to be removing the import of user-page-loader.js, do we?

Reply:
Apparently when I was trying to commit I had that line selected and I typed "git c" in the IDE instead the in the terminal 😭.
On the other hand I didn't change anything on that document. I believe that line 24 "<script src="/js/user-page-loader.js"></script>" is necessary ???

Reply:
Lol I hate when that happens, I have definitely done this before.

Yepp, it's necessary. Thanks for adding it back.